### PR TITLE
Windows compatibility, package build

### PR DESCRIPTION
**Pull request by SvyatSlav**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+dist
```
Comment (KirillTemnov) [inline .gitignore]: \n

File: package.json [hunk 2]
```diff
@@ -27,7 +25,10 @@
   "homepage": "https://github.com/driveback/utils#readme",
   "dependencies": {
     "async": "^2.5.0",
-    "js-cookie": "^2.1.4"
+    "cpy": "^7.0.1",
+    "cpy-cli": "^2.0.0",
+    "js-cookie": "^2.1.4",
+    "rimraf": "^2.6.2"
```
Comment (KirillTemnov) [inline package.json]: devDependencies?

File: package.json [hunk 3]
```diff
@@ -4,16 +4,14 @@
   "description": "Driveback Utils",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dist": "babel ./src --optional runtime --ignore node_modules --out-dir ./dist",
-    "prepublish": "npm run dist",
-    "postinstall": "mv dist/* ./ && rm -rf dist"
+    "build": "rimraf dist/ && babel ./src --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
+    "dist":"npm run build",
+    "prepare": "npm run build",
+    "postinstall": "cpy \"dist/*\" . && rimraf dist"
```
Comment (KirillTemnov) [inline package.json]: prepublish missed

Comment (SvyatSlav) [reply package.json]: Deprecated

**Pull request merged**